src/codecup.css

```diff
@@ -71,4 +71,65 @@
 	padding:5px;
 	display: flex;
 	justify-content: space-between;
-}
\ No newline at end of file
+}
+
+.editorjs-codeCup_languageSelectContainer {
+	position: relative;
+}
+
+.editorjs-codeCup_languageDropdown {
+	/* top:10px; left:100px; */
+	position: fixed;
```

Comment:
With `position:fixed` the dropdown is cropped off in mobile devices.  Would you be able to retain my changes?
```css
	position: absolute;
	top: 100%;
	left: 0;
```

<img width="318" alt="image" src="https://github.com/user-attachments/assets/b97d6ddf-5c51-4a1a-bee0-918793ace298" />